### PR TITLE
transition to remote theme

### PR DESCRIPTION
**Pull request by angfl97**
transitioned to the  remote [Jekyll-LLNL-Theme](https://github.com/LLNL/Jekyll-LLNL-Theme)

note: pull request [new page layouts](https://github.com/LLNL/Jekyll-LLNL-Theme/pull/15) needs to be approved first before changes made here will work properly

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: "LLNL Software Portal"
 author: "LLNL Open Source <open-source@llnl.gov>"
 title: LLNL Software Portal
 description: "Lawrence Livermore National Laboratory Software Portal"
+remote_theme: LLNL/Jekyll-LLNL-Theme
```
Comment (IanLee1521) [inline _config.yml]: Do we not also need the plugin listed in this file?

File: Gemfile [hunk 2]
```diff
@@ -5,4 +5,4 @@ gem 'jekyll-sitemap'
 gem 'jekyll-redirect-from'
 gem 'jekyll-seo-tag'
 gem 'jekyll-coffeescript'
-gem 'github-pages'
+gem 'github-pages', group: :jekyll_plugins
```
Comment (IanLee1521) [inline Gemfile]: Does this include the remote theme? I didn’t realize it did.

File: _config.yml (hunk 1 above)
Comment (angfl97) [reply _config.yml]: The README for the [LLNL remote theme](https://github.com/LLNL/Jekyll-LLNL-Theme) doesn't mention the Jekyll remote theme plugin. I'll make a new commit to incorporate the plugin as you did with the dev.llnl.gov site

File: Gemfile (hunk 2 above)
Comment (angfl97) [reply Gemfile]: according to the docs, all this line does is 

"Note: As of Jekyll 3.2, the default Jekyll site contains a Gemfile that locks Jekyll to the Gem version you build it with. To instead lock it to the version used by GitHub Pages, you'll uncomment the gem "github-pages", group: :jekyll_plugins line."

File: _config.yml (hunk 1 above)
Comment (IanLee1521) [reply _config.yml]: ok, I mean if it worked, then I wasn't going to worry about it, but I'd thought it needed to be there.

**Pull request merged**